index.js

```diff
@@ -0,0 +1,52 @@
+function getRandomIntInclusive(min, max) {
+	min = Math.ceil(min);
+	max = Math.floor(max);
+	return Math.floor(Math.random() * (max - min +1)) + min;
+}
+
+// console.log(getRandomIntInclusive(1,6));
+
+const dice = {
+	value : 1,
```

Comment:
You can actually assign the initial value of this dice to a random number as well using the `getRandomIntInclusive()` function. That way the dice will always start with a new random number, which will enable for an easier refresh.

Reply:
changing the value : 1 to , Math.random()